docs/intro/tutorials/gel_drizzle_booknotes.rst

```diff
@@ -0,0 +1,1264 @@
+.. _ref_guide_gel_drizzle_booknotes:
+
+====================================================
+Build a Book Notes App with Drizzle
+====================================================
+
+:edb-alt-title: Building a book notes app using Gel, Drizzle ORM, and Next.js
+
+In this tutorial we're going to walk you through building a Book Notes application
+that lets you keep track of books you've read along with your personal notes.
+We'll be using Gel as the database, Drizzle as the ORM layer, and Next.js as our
+full-stack framework.
+
+Gel is a data layer designed to help you figure out storage in your application
+*fast*. Drizzle is a TypeScript ORM that offers type safety and a great developer
```

Comment:
Maybe:

"Gel is a data layer designed to supercharge PostgreSQL with a graph-like object model, access control, Auth, and many other features."

I think it's also important here to explain why you'd want to use Drizzle + Gel right here. Basically with Gel you get access to EdgeQL (n+1 gone), schema, Auth, automatic embeddings, superb experience in projects with multiple languages where Gel schema/tooling is the same everywhere, etc. But you can also use Drizzle with it if you want or that's how you want to start learning Gel!